Assets/Scenes/LevelSelectScene.unity

```diff
@@ -0,0 +1,209 @@
+%YAML 1.1
```

Comment:
I think this scene should be renamed to `LevelSelect.unity` so that the pause menu level select button works. Alternatively, the scene name used for that button could be changed to "LevelSelectScene".